jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -380,6 +399,17 @@ static AbsoluteUnixPath getAppRootChecked(
     }
   }
 
+  @VisibleForTesting
+  static ContainerizingMode getContainerizingModeChecked(RawConfiguration rawConfiguration)
+      throws InvalidContainerizingModeException {
+    String mode = rawConfiguration.getContainerizingMode();
+    try {
+      return ContainerizingMode.valueOf(mode.toUpperCase(Locale.US));
```

Comment:
So you want to allow _PaCkAgEd_ and _eXPLODEd_?

Reply:
Haha, yeah. We won't document it. I think it's a good tradeoff to keep the code simple.